lua/lspconfig/server_configurations/terraform_lsp.lua

```diff
@@ -3,7 +3,7 @@ local util = require 'lspconfig.util'
 return {
   default_config = {
     cmd = { 'terraform-lsp' },
-    filetypes = { 'terraform', 'hcl' },
+    filetypes = { 'terraform', 'hcl', 'tf' },
```

Comment:
This is wrong. If filetype=tf that means the detection failed: https://github.com/neovim/neovim/blob/5e5b004da44c7075ed1e20ae9d05ab09c6f2ac58/runtime/lua/vim/filetype/detect.lua#L1524

filetype=tf is not terraform.